jwplayer-appletv-web-app/resources/configs/VCyJXbpY/config.json

```diff
@@ -1,11 +1,25 @@
 {
   "playlists": [
-    "K6Sl8yPJ",
-    "BXatQw7p",
-	  "E12RS6r7"
+    {
+      "playlistId": "K6Sl8yPJ",
+      "autoAdvance": true,
```

Comment:
@pablos1 It will be a little less flexible then. 
One thing that I would want to keep is an array of objects, instead of strings for the playlist configuration option, with the idea in mind that in the future we want to be able to set configuration options on a playlist basis.